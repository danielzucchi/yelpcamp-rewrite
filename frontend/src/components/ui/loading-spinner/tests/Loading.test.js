import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../Loading';

describe('Loading spinner', () => {
  it('When all props are present', () => {
    const wrapper = shallow(<Loading />);

    expect(wrapper.find('.loading')).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
