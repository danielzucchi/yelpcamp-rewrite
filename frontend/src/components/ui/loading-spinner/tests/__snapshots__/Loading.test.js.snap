// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading spinner When all props are present 1`] = `ShallowWrapper {}`;
